packages/amazonq/test/e2e_new/amazonq/helpers/pinContextHelper.ts

```diff
@@ -119,22 +106,13 @@ export async function getSubMenuItems(webview: WebviewView): Promise<{ items: We
  * NOTE: To find all possible text labels, you can call getPinContextMenuItems
  */
 export async function clickSubMenuItem(webview: WebviewView, itemName: string): Promise<void> {
-    const menuList = await waitForElement(webview, By.css('.mynah-detailed-list-items-block'))
-    await sleep(100)
-    const menuListItems = await menuList.findElements(By.css('.mynah-detailed-list-item.mynah-ui-clickable-item'))
-    for (const item of menuListItems) {
-        const textWrapper = await item.findElement(
-            By.css('.mynah-detailed-list-item-text.mynah-detailed-list-item-text-direction-row')
+    await sleep(0)
+    const item = await waitForElement(
+        webview,
+        By.xpath(
+            `//div[contains(@class, 'mynah-detailed-list-item') and contains(@class, 'mynah-ui-clickable-item')]//div[contains(@class, 'mynah-detailed-list-item-name') and text()='${itemName}']`
```

Comment:
these strings are gnarly, how does one find this? Are they querying the css based on these attributes?

Reply:
Wow cool! Maybe in the future we can build/find a nicer interface on top of these queries? This looks good for now though, nice find!